flake.nix

```diff
@@ -25,7 +25,6 @@
               pkgs.openssl
               openjdk17
               sbt
-              cmake
```

Comment:
This causes a bug with parallel building and syntax errors, so not worth having for now